tests/python-gpu/conftest.py

```diff
@@ -41,3 +41,16 @@ def local_cuda_cluster(request, pytestconfig):
 
 def pytest_addoption(parser):
     parser.addoption('--use-rmm-pool', action='store_true', default=False, help='Use RMM pool')
+
+def pytest_collection_modifyitems(config, items):
+    if config.getoption('--use-rmm-pool'):
+        blocklist = [
+            'python-gpu/test_gpu_with_dask.py::TestDistributedGPU::test_gpu_hist',
```

Comment:
We should keep this one.

Reply:
Do we need to run this one with RMM? This is quite expensive test.

Reply:
Actually we can put it back, if you strongly feel we should keep it.

Reply:
Thanks for working on this.  Let's put it back for now.  It's a full test for training.  Also it might be useful if we want to log the memory consumption.